src/locales/de.json.ts

```diff
@@ -0,0 +1,332 @@
+import { exportAsMessages } from './helpers';
+
+exportAsMessages('_locales/en/messages.json', {
```

Comment:
The filename should be `_locales/de/messages.json`.

Reply:
Sorry, missed this.